### PR TITLE
Update start a private network tutorial

### PR DESCRIPTION
**Pull request by obxium**
- Minor content updates
- Updated screenshots

- [x] Are the audience and objective of the document clear? E.g. a document for developers that should teach them about transaction fees.
- [x] Is the writing:
  - [x] Clear: No jargon.
  - [x] Precise: No ambiguous meanings.
  - [x] Concise: Free of superfluous detail.
- [ ] Does it follow our style guide?
- [ ] If this is a new page, does the PR include the appropriate infrastructure, e.g. adding the page to a sidebar?
- [ ] Build the page ($ cd website && yarn start). Does it render properly? E.g. no funny lists or formatting.
- [ ] Do links go to rustdocs or devhub articles rather than code?
- [ ] If this PR addresses an issue in the queue, have you referenced it in the description?

### CODE REVIEW

File: docs/tutorials/start-a-private-network/alicebob.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Alice (or whomever is playing her) should run these commands from node-template
   --chain local \
   --alice \
   --port 30333 \
-  --ws-port 9945 \
```
Comment (danforbes) [inline docs/tutorials/start-a-private-network/alicebob.md:32]: I think other parts of this tutorial expect the `9945` port, e.g.
> In your web browser, navigate to https://polkadot.js.org/apps/#/settings?rpc=ws://127.0.0.1:9945.

This was changed very recently and personally I prefer it "the old way" (the way you have it here) but it needs to be consistent across the whole tutorial, of course. Also, if Alice is using `9944` I would think Bob should use `9945` and I think right now he's configured to use `9946`.

**Review by danforbes — CHANGES_REQUESTED** [review]
Thank you for these improvements but I think there need to be a few small changes for consistency.

File: docs/tutorials/start-a-private-network/alicebob.md (hunk 1 above)
Comment (obxium) [reply docs/tutorials/start-a-private-network/alicebob.md:32]: My bad! I thought I'd accounted for all the port changes. I just corrected that and the example output to match.

**Review by riusricardo — APPROVED** [review]
Thanks @obxium! I pushed a few small changes. The rest LGTM.

**Review by wheresaddie — APPROVED** [review]
:+1:

**Review by wheresaddie — APPROVED** [review]

**Review by danforbes — APPROVED** [review]
Thank you!

**Pull request merged**